test/sqllogictest/with_mutually_recursive.slt

```diff
@@ -155,6 +155,15 @@ SELECT (SELECT COUNT(*) FROM foo) FROM bar;
 2
 2
 
+## Test that we can determine recursively defined collections are empty
+query I
+WITH MUTUALLY RECURSIVE
+    foo (a int) AS (SELECT * FROM foo UNION SELECT 1),
+    bar (b int) AS (SELECT a + b FROM foo, bar)
+SELECT * FROM bar;
+----
+
+
```

Comment:
I need to fix this. It was originally meant to be an `EXPLAIN`, but now it also turns out that we delete a bunch of other test queries, any one of which could be kept as a better test than this.